src/docs/cookbook-emails/set-up-outgoing-email-settings/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Set up outgoing email settings
+slug: set-up-outgoing-email-settings
+updated: 2020-12-02
+category: cookbook-emails
+ingress:
+  This article describes how to set up a custom email sender address for
+  your production marketplace.
+published: true
+---
+
+Setting up reliable email delivery is crucial for your marketplace to
+work properly. It is also important to brand your email sending name and
+address to look professional and polished.
+
+Flex uses [SendGrid](https://sendgrid.com/) as an email service
+provider. We manage your SendGrid account and settings automatically,
+and the costs are included in your Flex subscription.
+
+When you have your production environment created and you have bought
+your custom domain for it, you should set up and validate the outgoing
+email settings. Follow the steps below, and you can be sure that your
+emails will be delivered reliably.
+
+## 1. Add sender name and address
+
+Make sure you select your production environment in the top navigation
+in Console. Then head on to the General settings page in the Build
```

Comment:
We can link to the page, but it will result in a redirect to the marketplace they had last selected.

To avoid confusion with a different env/org, I left this out. They really only need to click on the "Build" link in the top to go to the page. Do you still think it makes sense to give the link?

Reply:
Not necessary, this was more of a suggestion that is probably not good.